src/main/kotlin/no/nav/familie/ef/iverksett/iverksetting/IverksettingRepository.kt

```diff
@@ -78,6 +75,17 @@ class IverksettingRepository(val namedParameterJdbcTemplate: NamedParameterJdbcT
                ?: error("Finner ikke iverksett med behandlingId=${behandlingId}")
     }
 
+    fun hentAlleBehandlinger(): List<Iverksett> {
+        val resultSetExtractor = ResultSetExtractor { rs ->
+            val behandlinger = mutableListOf<Iverksett>()
+            while (rs.next()) {
+                behandlinger.add(rs.getJson<Iverksett>("data") ?: error("Kunne ikke transformere iverksett data"))
+            }
+            behandlinger
+        }
+        return namedParameterJdbcTemplate.query(HENT_ALLE_IVERKSETT_SQL, resultSetExtractor) ?: error("Fant ingen behandlinger")
+    }
```

Comment:
Hvis man bruker `RowMapper` i stedet, så trenger man ikke forholde seg til en mutableList eller `rs.next()`
```suggestion
    fun hentAlleBehandlinger(): List<Iverksett> {
        return namedParameterJdbcTemplate.query(HENT_ALLE_IVERKSETT_SQL) { rs, _ ->
            rs.getJson<Iverksett>("data")
        }
    }
```